src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.Scanner;
+
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Generates a greeting
+        greeting();
+        request();
     }
-}
+
+    public static void request(){
+        Scanner in = new Scanner(System.in);
+        Task[] list = new Task[100];
+        String line;
+        int i=0,j;
+
+        while(true) {
+            String status ="[✗] ";
+            line = in.nextLine();
```

Comment:
instead of line, i suggest "userInput" to let people understand that the line variable is suppose to contain the user input.